sdk/communication/azure-communication-sms/README.md

```diff
@@ -54,23 +54,27 @@ endpoint = os.getenv('AZURE_COMMUNICATION_SERVICE_ENDPOINT')
 sms_client = SmsClient(endpoint, DefaultAzureCredential())
 ```
 
-### Sending an SMS
+### Sending SMS Messages
 
 Once the client is initialized, the `.send()` method can be invoked:
 
 ```Python
-from azure.communication.sms import SendSmsOptions, PhoneNumberIdentifier
+from azure.communication.sms import SendSmsOptions
 
-smsresponse = sms_client.send(
+sms_responses = sms_client.send(
     from_phone_number=PhoneNumberIdentifier("<leased-phone-number>"),
-    to_phone_number=[PhoneNumberIdentifier("<to-phone-number>")],
+    to_phone_numbers=["<to-phone-number-1>", "<to-phone-number-2>", "<to-phone-number-3>"],
     message="Hello World via SMS",
-    send_sms_options=SendSmsOptions(enable_delivery_report=True)) # optional property
+    enable_delivery_report=True, # optional property
```

Comment:
I guess I am confused now, enableDeliveryReport is not optional anymore, right?

Reply:
It's optional to the client, but we'll default to ```false``` if its not provided.